diagnostic.md

```diff
@@ -62,7 +62,7 @@ var result = strangeAdd(0, -5);
 
 What is the value of result?  Explain your answer.
 
-_Replace this text with your answer._
+unsure..
 
```

Comment:
Incorrect. That's okay.

The value of result is 5._
a is 0 and and therefore not greater than 0.  b is -5 and also not greater than 0.  Therefore, a - b will be returned.  0 - (-5) is equal to 5.

always feel free to test code snippets like this in node or a repl